doc/md/fundamentals/2-types/13-stable-types.md

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 13
+---
+
+# Stable types
```

Comment:
All of the examples here, though wrapped in actor, don't indent the their bodies. 
Unless @jessiemongeon1 instructed you otherwise, I would indent the bodies by two spaces.